src/main/java/com/netgrif/application/engine/elastic/domain/ElasticTaskJob.java

```diff
@@ -0,0 +1,20 @@
+package com.netgrif.application.engine.elastic.domain;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+public class ElasticTaskJob {
+
+    private ElasticJob typeJob;
+
+    private ElasticTask task;
+
```

Comment:
In the `QueueManager` a method that is often called on the `ElasticTaskJob` instance is `.getTask().getTaskId()`. Why not add a public method `getTaskId()` that would hide this chained call?

Reply:
Done